doc/modules/preprocessing.rst

```diff
@@ -25,7 +25,9 @@ Standardization, or mean removal and variance scaling
 **Standardization** of datasets is a **common requirement for many
 machine learning estimators** implemented in scikit-learn; they might behave
 badly if the individual features do not more or less look like standard
-normally distributed data: Gaussian with **zero mean and unit variance**.
+normally distributed data: Gaussian with **zero mean and unit/factor two standard**
+**deviation** in reference to **Gelman, A. (2008). Scaling regression inputs by** 
+**dividing by two standard deviations. Statistics in medicine, 27(15), 2865-2873**.
```

Comment:
I think you can add a reference bellow the document as

```
.. [1] Gelman, A. (2008). Scaling regression inputs by
dividing by two standard deviations. Statistics in medicine, 27(15), 2865-2873
```

and refer to it in the document.